dockers/ci-runner-hpu/start.sh

```diff
@@ -93,4 +93,4 @@ trap 'cleanup; exit 143' TERM
 
 # To be aware of TERM and INT signals call run.sh
 # Running it with the --once flag at the end will shut down the agent after the build is executed
-./run.sh --once & wait $!
+./run.sh & wait $!
```

Comment:
How is this different from running the script?

Also the comment above is outdated

Reply:
Will you do the change in a different PR?